resources/zib/zib-HeartRate.xml

```diff
@@ -0,0 +1,207 @@
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.12.3.2--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-heartrate-v3.4-2020EN" />
+        <map value="NL-CM:12.3.6" />
+        <comment value="HeartRateMeasurementMethod" />
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="code" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.component:heartbeatRegularity">
+      <path value="Observation.component" />
+      <sliceName value="heartbeatRegularity" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:heartbeatRegularity.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="301112006" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:heartbeatRegularity.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="HeartbeatRegularity" />
+      <definition value="Regularity of the heartbeat." />
+      <alias value="HartslagRegelmatigheid" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.12.3.1--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-heartrate-v3.4-2020EN" />
+        <map value="NL-CM:12.3.5" />
+        <comment value="HeartbeatRegularity" />
+      </mapping>
+    </element>
+    <element id="Observation.component:interpretationHeartRate">
+      <path value="Observation.component" />
+      <sliceName value="interpretationHeartRate" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:interpretationHeartRate.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="301113001" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:interpretationHeartRate.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="InterpretationHeartRate" />
+      <definition value="Interpretation heart rate" />
+      <alias value="InterpretatieFrequentie" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <mapping>
+        <identity value="zib-heartrate-v3.4-2020EN" />
+        <map value="NL-CM:12.3.7" />
+        <comment value="InterpretationHeartRate" />
+      </mapping>
+    </element>

```

Comment:
Missing the "InterpretationHeartRateCodelist" valueSet binding

Reply:
Klopt, gefixt!